app/assets/stylesheets/home.css.scss

```diff
@@ -58,6 +58,35 @@ body {
   margin-bottom: 15px;
 }
 
+.principal-button {
+  text-transform: uppercase;
+  background-color: #000;
+  color: white;
+  font-size: 10px;
+  font-weight: 700;
+  border: none;
+  margin-top: 30px;
+}
+
+.principal-field, .principal-button {
+  height: 40px;
+  width: 200px;
+  border: 1px solid #585858;
+}
+
+.principal-field {
+  margin: 0 0 5px 0;
+  text-align: center;
+  font-size: 15px;
+}
+
+hr {
+  width: 100px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  border-top: 1px solid #585858;
```

Comment:
When you use the same colour in several properties, you can define a constant for it. You have to write it at the top:
```
$constante_name: #585858;` 
```
Then you can use it in the properties just writing its name:
```
.class {
  color: $constant_name;
}
```